qemu/tests/expose_host_mtu.py

```diff
@@ -82,7 +82,6 @@ def cleanup_ovs_ports(netdst, ports):
     vm.verify_alive()
 
     vm_iface = vm.get_ifname()
-    # TODO, will support windows later
     process.system_output(params["set_mtu_cmd"] % host_hw_interface)
```

Comment:
@xiywang  The host physical nic mtu is changed here, should recover it's original value(default 1500) after testing is finished?